chart/values.yaml

```diff
@@ -12,6 +12,7 @@ controllerNamespace: ""
 
 # "{\"nat\": \"true\"}"
 jobNodeSelector: ""
+jobBackoffLimit:
```

Comment:
A default value?

Reply:
The default is the maximum value of int, compatible with the original default value

Reply:
How about "" instead of just leaving it empty.

Reply:
I agree with your point of view.